src/main/ui/app/component/AddOnSearch.jsx

```diff
@@ -56,32 +56,45 @@ export default class AddOnSearch extends Component {
     }
 
     render() {
-        return (
-                <div>
-                    Type:
-                    <select onChange={event => this.setType(event.target.value)}>
-                        <option value="">All Types</option>
-                        <option value="OMOD">Module (OMOD)</option>
-                        <option value="OWA">Open Web App (OWA)</option>
-                    </select>
 
-                    <br/>
+const title = (
+<span>Type: {this.state.type}</span>
```

Comment:
Actually we should change this so it's displaying as one of these strings:

* All Types
* Modules
* Open Web Apps